### PR TITLE
Misc changes from Reliagate Integration

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

File: modules/probe.sh [hunk 1]
```diff
@@ -195,7 +195,12 @@ probe_kernel_in_boot_and_root() {
   done
 
   if [ -z "${kernel_imagetype_path}" ]; then
-    log_warn "Unfortunately we where not able to find the Linux kernel image."
+    log_warn "Unfortunately we were not able to find the Linux kernel image."
+    log_fatal "Please specify the image name using MENDER_GRUB_KERNEL_IMAGETYPE"
+  fi
+
+  if [ "$(echo "${kernel_imagetype_path}" | wc -w)" -gt "1" ]; then
+    log_warn "Unfortunately we were not able to find a unique Linux kernel image."
```
Comment (mirzak) [inline modules/probe.sh:203]: There is a separate [PR](https://github.com/mendersoftware/mender-convert/pull/212) to fix this, so would just drop this changeset

File: modules/disk.sh [hunk 2]
```diff
@@ -96,7 +96,7 @@ disk_align_sectors() {
 # $2 - destination file
 # $3 - offset in number of 512 sectors
 disk_write_at_offset() {
-  run_and_log_cmd "dd if=${1} of=${2} seek=${3} conv=sparse,notrunc status=none"
+  run_and_log_cmd "dd if=${1} of=${2} seek=${3} conv=notrunc status=none"
```
Comment (mirzak) [inline modules/disk.sh:99]: I do not understand how this would cause problems? Can you give an example?

File: configs/generic_x86-64_hdd_config [hunk 3]
```diff
@@ -39,6 +39,9 @@ MENDER_STORAGE_TOTAL_SIZE_MB=16000
 # Nothing to copy
 MENDER_COPY_BOOT_GAP="n"
 
+# Not using UBoot.
+MENDER_GRUB_EFI_INTEGRATION="y"
+
```
Comment (mirzak) [inline configs/generic_x86-64_hdd_config:44]: This is the default value, as specified in,

https://github.com/mendersoftware/mender-convert/blob/7b6d957170236f139e5022d0d7d74a726ad6bae3/configs/mender_convert_config#L175

Do not see a huge benefit of being explicit about it here.

File: mender-convert-modify [hunk 4]
```diff
@@ -120,7 +120,7 @@ if [ "${MENDER_GRUB_EFI_INTEGRATION}" == "y" ]; then
   if [ -z "${MENDER_GRUB_INITRD_IMAGETYPE}" ]; then
```
Comment (mirzak) [inline mender-convert-modify:120]: Missing `Changelog:`

File: modules/probe.sh (hunk 1 above)
Comment (drewmoseley) [reply modules/probe.sh:203]: Yeah, I saw that over the weekend and like the other solution better.

File: configs/generic_x86-64_hdd_config (hunk 3 above)
Comment (drewmoseley) [reply configs/generic_x86-64_hdd_config:44]: Dang. My wrapper script didn't include that config file.  Agreed this is unneeded.

File: modules/disk.sh (hunk 2 above)
Comment (drewmoseley) [reply modules/disk.sh:99]: I didn't understand it either but it definitely did make a difference.  Last week when I tested it. ;-)
Now for some reason it doesn't seem to be needed. Who knows.

**Comment by drewmoseley** [comment]
Updated as discussed.

**Pull request merged**